.github/workflows/periodic-azure-clean.yml

```diff
@@ -0,0 +1,37 @@
+name: Periodic Azure cleanup
+
+on:
+  schedule:
+    - cron: "0 0 * * 1" # Every Saturday at midnight UTC, I assume the main build won't be triggered right before this one
+  workflow_dispatch: {}
+
+env:
+  MANUAL: ${{ github.event_name == 'workflow_dispatch' }}
+
+jobs:
+  azure-cleanup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: test if is maintainer
+        uses: tspascoal/get-user-teams-membership@v3
+        id: testUserGroup
+        if: ${{ env.MANUAL == 'true' }}
+        with:
+          username: ${{ github.actor }}
+          team: "maintainers"
+          GITHUB_TOKEN: ${{ secrets.GH_GROUPS_READ_TOKEN }}
```

Comment:
(optional) If this secret is a GitHub token for monabot, it could be worth noting it in our runbook https://github.com/monadahq/monada/blob/main/docs/secrets.md. Maybe we can consolidate some of the secrets used by our workflows

Reply:
Totally!